dotcom-rendering/src/components/marketing/epics/ContributionsEpicButtons.tsx

```diff
@@ -186,6 +188,7 @@ export const ContributionsEpicButtons = ({
 	const getChoiceCardCta = (cta: Cta): Cta => {
 		if (
 			showThreeTierChoiceCards &&
+			!variantOfChoiceCard &&
```

Comment:
is this if statement not redundant now? We pass the prop `variantOfChoiceCard` in Live blog and normal Epic so both cases will fall into the next if statement